server/models/UserToken.ts

```diff
@@ -71,6 +80,28 @@ UserToken.init(
       type: DataTypes.JSONB,
       allowNull: true,
     },
+    scope: {
+      type: DataTypes.ARRAY(
+        DataTypes.ENUM(
+          'email',
+          'account',
+          'expenses',
+          'orders',
+          'transactions',
+          'virtualCards',
+          'payoutMethods',
+          'paymentMethods',
+          'host',
+          'root',
+          'conversations',
+          'updates',
+          'webhooks',
+          'applications',
+          'connectedAccounts',
+        ),
+      ),
```

Comment:
:ice_cream: Doing that in all models will reduce the number of places to update when editing the scopes, reducing the risk of forgetting one


```suggestion
      type: DataTypes.ARRAY(DataTypes.ENUM(Object.values(OAuthScopes))),
```